csp/impl/wiring/signature.py

```diff
@@ -134,6 +213,56 @@ def parse_inputs(self, forced_tvars, *args, allow_subtypes=True, allow_none_ts=F
 
         return tuple(type_resolver.ts_inputs), tuple(type_resolver.scalar_inputs), type_resolver.tvars
 
+    def _parse_inputs_pydantic(self, forced_tvars, *args, allow_none_ts=False, **kwargs):
+        from csp.utils.object_factory_registry import Injected
```

Comment:
Didnt realize this object factory still exists here, we should open an issue to ensure its no longer used and remove it
Its possible some parts of it are still used for memoization but it was written to handle a lot more than that, but never got used

Reply:
Opened #377